roles/schema_registry/tasks/main.yml

```diff
@@ -142,9 +142,19 @@
   tags:
     - configuration
 
-- name: Configure RBAC
+- name: Configure RBAC for OAuth User
   include_tasks: rbac.yml
-  when: rbac_enabled|bool
+  vars:
+    sr_user: "{{schema_registry_oauth_user}}"
+  when: rbac_enabled|bool and oauth_enabled
```

Comment:
just for consistency here maybe we can have oauth_enabled|bool

Reply:
updated